build.rs

```diff
@@ -16,5 +16,7 @@ fn main() {
         "has_to_int_unchecked",
     );
 
+    ac.emit_path_cfg("1.trailing_ones()", "has_leading_trailing_ones");
```

Comment:
This fails even on current stable -- see `cargo check -vv`. This needs to be an expression check using an explicit integer type to pass.

Reply:
Ah, whoops. Good catch.